src/components/Table/Table.types.ts

```diff
@@ -69,7 +69,7 @@ export type CustomAction<RecordType extends GenericRecord> = {
   icon: ReactNode,
   isDanger?: boolean,
   isDisabled?: (record: RecordType, index?: number) => boolean | boolean,
-  onClick: (record: RecordType, index: number | undefined, event: React.MouseEvent<Element, MouseEvent>) => void,
+  onClick?: (record: RecordType, index: number | undefined, event: React.MouseEvent<Element, MouseEvent>) => void,
```

Comment:
does it make sense to have an action without the action logic?

Reply:
yes, because edit and delete actions could be defined directly as table props, but i think that is better force to use onClick if some custom edit/delete actions are defined and so ignore the onEditRow/onDeleteRow props in this case.

Reply:
The on click priorityis exactly how you describe it